### PR TITLE
Create organization type selector component

### PR DESCRIPTION
**Pull request by JamesSCBD**

### CODE REVIEW

File: components/controls/selectors/OrganizationTypeSelector.vue [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+          .map((value) => this.organizationTypesOptions
+            .find((option) => option.OrganizationTypeUID === value) || { OrganizationTypeUID: value, isMissing: true });
+      },
+      set(orgTypes) {
+        const ids = this.asArray(orgTypes).map((orgType) => orgType.OrganizationTypeUID);
+        this.$emit('input', this.multiple ? ids : ids[0]);
+      },
+    },
+    ...mapGetters({
+      countryOptions: 'countries/list',
```
Comment (sbilodeau) [inline components/controls/selectors/OrganizationTypeSelector.vue:69]: Please remove country list

File: components/controls/selectors/EventSelector.vue [hunk 2]
```diff
@@ -55,11 +55,11 @@ export default {
   computed: {
     selectedEvents: {
       get() {
-        return this.asArray(this.vModel).map((value) => this.events.find((option) => option.EventUID === value) || { EventUID: value, isMissing: true });
+        return this.asArray(this.value).map((value) => this.events.find((option) => option.EventUID === value) || { EventUID: value, isMissing: true });
```
Comment (sbilodeau) [inline components/controls/selectors/EventSelector.vue:58]: I’m not sure but If multiple is false, you *cannot* pass an array to multiselect

File: components/controls/selectors/OrganizationTypeSelector.vue [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+      required: true,
+      default() {
+        return [];
+      },
+    },
+  },
+  computed: {
+    selectedOrganizationTypes: {
+      get() {
+        return this.asArray(this.value)
```
Comment (sbilodeau) [inline components/controls/selectors/OrganizationTypeSelector.vue:59]: Same as above, If multiple is false, you *cannot* pass an array to multiselect

File: components/controls/selectors/EventSelector.vue (hunk 2 above)
Comment (JamesSCBD) [reply components/controls/selectors/EventSelector.vue:58]: Actually we shouldn't pass a array if multiple is false. We pass a only string (single value) then return type is also string. I tested on my side this line of code work fine in both condition

File: components/controls/selectors/OrganizationTypeSelector.vue (hunk 1 above)
Comment (JamesSCBD) [reply components/controls/selectors/OrganizationTypeSelector.vue]: Removed

File: components/controls/selectors/EventSelector.vue (hunk 2 above)
Comment (sbilodeau) [reply components/controls/selectors/EventSelector.vue:58]: I think you a confused my commeny:

`selectedEvents` Is for the multiselect component not the page that use the EventSelecter.  When a multiple on a vue-multiselect is false, le vue-multiselect accept and return an Event object (not an array). That mean that selectEvents getter (that set he value to the vue-multiselect) should be an Event object.  

Please test the following: put one event in the quweystring of the contact serach, chnage the multiple to false, rebuld, reload and see if the metting is correctly selected in the selector

Comment (JamesSCBD) [reply components/controls/selectors/EventSelector.vue:58]: > Please test the following: put one event in the quweystring of the contact serach, chnage the > multiple to false, rebuld, reload and see if the metting is correctly selected in the selector

I have tested this it is working fine with given scenario. Let me know if you still want to change it

Comment (sbilodeau) [reply components/controls/selectors/EventSelector.vue:58]: Ok i will test it today

Comment (sbilodeau) [reply components/controls/selectors/EventSelector.vue:58]: yes it's works!!! Great make thing simpler!

**Pull request merged**